### ISSUE TITLE
Welcome bot will not let me add a message to private channels

### ISSUE THREAD

**Issue by DHaussermann** [mattermost/mattermost-plugin-welcomebot#72]
Seems like the feature of adding a channel level welcome message should work for both public and private channels. 
When I try using this feature I see a message saying this is not supported for DM channels. However the channel is not a DM.
![image](https://user-images.githubusercontent.com/35588783/120028233-6a206280-bfc2-11eb-985c-64b9139e26e5.png)

Steps:
- Create or open any private channel 
- Type `/welcomebot set_channel_welcome "I like pickles"`
**Observed**: `welcome messages are not supported for direct channels`

**Comment by hanzei** [comment]
It works exactly like the code dictates it: https://github.com/mattermost/mattermost-plugin-welcomebot/blob/29c0079329652a42c852ade6aa9309359777f22b/server/command.go#L133-L136

**Comment by jasonsmithtesla** [comment]
I was surprised to run into this (why shouldn't it work in a private channel?) and confused by the misleading error message.

**Comment by hanzei** [comment]
Yes, the code is wrong. Only direct channels should be prohibited, not private ones.

**Comment by mickmister** [comment]
Reopening since the fix PR has been reverted https://github.com/mattermost/mattermost-plugin-welcomebot/pull/109
